Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.py

```diff
@@ -687,6 +687,7 @@ def main():  # pragma: no coverage
             demisto.info(f'ad-authenticate-and-roles command - entry results: {entry_result}')
             return_results(entry_result)
         else:
+            demisto.debug('testtttttt')
```

Comment:
@esharf -  I left this here because I want you to be able to see the different 'unit Testing and Lint' jobs I've created. 
With this new debug log, the lint is triggered for the OpenLDAP integration and you can see that for each job the relevant docker image is selected.

Reply:
don't forget to remove

Reply:
Removed :) Thanks.